plumex/config.py

```diff
@@ -71,8 +78,38 @@
         "917": (400, 1679),
         "919": (500, 2453),
         "920": (300, 2094),
+        "nowind": (200, 372),
+        "wind": (130, 411),
+        "lo1": (900, 1400),
+        "hi1": (1000, 1500),
+        "hi2": (700, 1100),
+    },
+    "gauss_space_kws": {
+        "off": {"kernel_size": (1, 1), "sigma_x": 1e-1},
```

Comment:
```suggestion
        "off": {"kernel_size": (1, 1), "sigma_x": 1e-1, "sigma_y":1e-1},
```
By "off" do you mean no blurring applied? Or just specifically in the `x`-direction? If the former then I think you need to also specify the extra `sigma_y` value because otherwise it will run default value of 15.

Also do you think the `apply_gauss_space_blur`/ `apply_gauss_time_blur` functions should have default values assigned?

Reply:
off = no blurring.  You're right - the default args of `apply_gauss_space_blur` will set it to 15.

I think ideally, that function wouldn't have default values, or else it should mimic `cv2.GaussianBlur` behavior: a default `sigmaY` based on `sigmaX` (which is what i was doing by leaving `sigma_y` out of the `off` dictionary.

Reply:
I also cleared out the named variants for these in `plume_points_variant_gauss_space_kws` table in matrix:/home/plumes/trials/center/plume_points.db